pydo/database/sqlite.py

```diff
@@ -0,0 +1,53 @@
+from .database import Database
+import sqlite3
+from os import path
+from ..config.config import dump_config
+
+
+class SqliteDatabase(Database):
+    db_ext = '.db'
+
+    def __init__(self, name, db_path, config_path=None):
+        super().__init__(name, db_path, config_path)
+        self.cursor = None
+        self.connection = None
+
+        if self.name.endswith(SqliteDatabase.db_ext):
+            self.complete_path = path.join(self.path, self.name)
+        else:
+            self.complete_path = path.join(self.path,
```

Comment:
Minor but there's a lot of duplication across this if/else. And in py3 it's best to use pathlib instead of string manipulation for paths, e.g. https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.with_suffix